<?xml version="1.0"?>
<launch>
    <include file="$(find ros_simulation)/launch/sim_w1.launch" />
    
    <param name="des_pos_x" value= "0.0" />
    <param name="des_pos_y" value= "1.0" />
    <param name="publish_frequency" type="double" value="1.0" />

    <node pkg="ros_simulation" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="ros_simulation" type="go_to_point_service.py" name="go_to_point" />
    <node pkg="ros_simulation" type="bug_as.py" name="bug_action_service" output="screen" />  
    
    <node pkg="ros_simulation" type="info_node.py" name="info" output="screen" launch-prefix="xterm -hold -e"/>
    <node pkg="ros_simulation" type="goals_srv.py" name="goals_srv"  />
    <node pkg="ros_simulation" type="client.py" name="client" output="screen" launch-prefix="xterm -hold -e" />

</launch>
